App/Redux/PreferencesRedux.ts

```diff
@@ -78,7 +129,8 @@ export const PreferencesSelectors = {
   // TODO: Need typed state
   onboarded: (state: any) => state.preferences.onboarded,
   pending: (state: any) => state.preferences.pending,
-  profile: (state: any) => state.preferences.profile
+  profile: (state: any) => state.preferences.profile,
+  service: (state: any, name: ServiceType) => state.preferences.services[name]
```

Comment:
You can use `RootState` here and get all the TS goodness.

Reply:
done